src/env.js

```diff
@@ -1,3 +1,3 @@
 export default {
-    SPOTIFY_CLIENT_ID: 'a71dee236b8946cdab38e88e7ed0467e',
+    SPOTIFY_CLIENT_ID: '127702bf159a4f9e9e504f0952e6d105',
```

Comment:
At some point soon we should also store this as an environmental variable or exclude it from the repo and re-generate our client ids